extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/JacksonBuildTimeConfig.java

```diff
@@ -23,6 +23,12 @@
     @ConfigItem(defaultValue = "false")
     public boolean writeDatesAsTimestamps;
 
+    /**
+     * If enabled, Jackson will ignore case during Enum deserialization.
+     */
+    @ConfigItem(defaultValue = "false")
```

Comment:
Is this the Jackson default?

Reply:
Yes, this feature is disabled by default.